### PR TITLE
Prep 5.8.4

### PR DESCRIPTION
**Pull request by jburel**
* Bump dependencies
* Add changelog

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-insight-push#299](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/OMERO-insight-push/299/). See the [console output](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/OMERO-insight-push/299/consoleText) for more details.
Possible conflicts:
  - PR #333 jburel 'drop compatibility to test new Jenkins'
    - build.gradle
  - PR #377 sukunis 'Fix ontology loads'
    - build.gradle

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-insight-push#300](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/OMERO-insight-push/300/). See the [console output](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/OMERO-insight-push/300/consoleText) for more details.
Possible conflicts:
  - PR #333 jburel 'drop compatibility to test new Jenkins'
    - build.gradle
  - PR #377 sukunis 'Fix ontology loads'
    - build.gradle

--conflicts

**Comment by dominikl** [comment]
Looks good to me. Any idea why the build is failing? Also a bit worried that the merge-ci build doesn't work (strangely local gradle build does).

**Review by dominikl — APPROVED** [review]

**Comment by jburel** [comment]
the build was failing because artifacts.openmicroscopy.org was down due to the power outage

**Comment by jburel** [comment]
> Looks good to me. Any idea why the build is failing? Also a bit worried that the merge-ci build doesn't work (strangely local gradle build does).

@dominikl 
I have tested locally the following:
*  ``gradle build`` 
*  `` unzip build/distributions/OMERO.insight-5.8.4-SNAPSHOT.zip``
*  bash OMERO.insight-5.8.4-SNAPSHOT/bin/omero-insight 

Everything works as expected.
Then tested the latest release downloaded from GitHub and unzipped i.e. 
``bash ~/Desktop/OMERO.insight-5.8.3/bin/omero-insight ``
It works too

**Comment by pwalczysko** [comment]
> Everything works as expected.
> Then tested the latest release downloaded from GitHub and unzipped i.e.
> `bash ~/Desktop/OMERO.insight-5.8.3/bin/omero-insight `
> It works too

What @dominikl is meaning is (i.e. grab the build from jenkins, which has to be started using bash)

1. Download https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-insight-build/lastSuccessfulBuild/artifact/build/distributions/OMERO.insight-5.8.4-SNAPSHOT.zip and unzip it
2. Try to start it and fail with 

```
(cli) pwalczysko@Petrs-MBP ~ % bash ~/Downloads/OMERO.insight-5.8.4-SNAPSHOT
/Users/pwalczysko/Downloads/OMERO.insight-5.8.4-SNAPSHOT: /Users/pwalczysko/Downloads/OMERO.insight-5.8.4-SNAPSHOT: is a directory
(cli) pwalczysko@Petrs-MBP ~ % bash ~/Downloads/OMERO.insight-5.8.4-SNAPSHOT/bin/omero-insight
Error: LinkageError occurred while loading main class org.openmicroscopy.shoola.Main
	java.lang.ExceptionInInitializerError: null
```

**Comment by jburel** [comment]
It does not work from merge-ci sorry I forgot to mention it in my testing report

**Comment by dominikl** [comment]
👍 Any idea how we could fix the msi installer issue? https://github.com/ome/omero-insight/issues/400 . I guess that will be bigger effort to find out what's going wrong there, right?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+5.8.4 (January 2024)
```
Comment (dominikl) [inline CHANGELOG.md:1]:
```suggestion
5.8.4 (May 2024)
```

**Comment by dominikl** [comment]
Looks good. Just 'January' shoudl be updated to 'May' :-)

**Pull request merged**